lib/f00px/request.rb

```diff
@@ -28,5 +28,13 @@ def post(url, params = {})
       end
     end
 
+    def http(method, url, params = {})
```

Comment:
Could we name it `request` for consistency with existing helpers? (`get` and `post`)